workspace.go

```diff
@@ -112,6 +112,8 @@ type VCSRepo struct {
 	Identifier        string `json:"identifier"`
 	IngressSubmodules bool   `json:"ingress-submodules"`
 	OAuthTokenID      string `json:"oauth-token-id"`
+	RepositoryHTTPURL string `json:"repository-http-url"`
+	ServiceProvider   string `json:"service-provider"`
```

Comment:
I see `"github"` is [hard coded in the tests](https://github.com/hashicorp/go-tfe/pull/191/files#diff-7dc0e5057910da963c46a1e6954bb53b648aff3e8d5bcba8bc78c4aa01b5e4abR231). Does that mean that `ServiceProvider` is always `"github"`, or is there a set of known ServiceProvider types? 

If its the latter, what do you think about creating an explicit `ServiceProviderType` like we do for [RunStatus](https://github.com/hashicorp/go-tfe/blob/master/run.go#L49-L72).

Reply:
In the tests we assume github right now with `githubIdentifier`.